wsd/DocumentBroker.cpp

```diff
@@ -4960,6 +4972,41 @@ bool DocumentBroker::forwardUrpToChild(const std::string& message)
     return _childProcess && _childProcess->sendUrpMessage(message);
 }
 
+std::string DocumentBroker::applySignViewSettings(const std::string& message,
+                                                  const std::shared_ptr<ClientSession>& session)
+{
+    std::string finalMsg = message;
+    if (_isViewSettingsUpdated)
+    {
+        Poco::JSON::Object signatureJson;
+        LOG_INF("HACK: Try to create Object ");
```

Comment:
I imagine this is still a work in progress? This message will be worrying to see at INFO level in deployment.

Reply:
Same, Already removed it.